public/index-staging.html

```diff
@@ -2,35 +2,35 @@
 <html lang="en">
```

Comment:
This file can probably be fully removed, we aren't planning on a public testnet instance.